cypress/integration/app_spec.ts

```diff
@@ -22,9 +22,21 @@ function setupServerMocks() {
     "https://freesound.org/data/previews/462/462808_8386274-lq.mp3",
     { fixture: "sound.mp3" }
   );
-  cy.intercept("GET", "https://freesound.org/apiv2/search/text/*", {
-    fixture: "search-results.json",
-  });
+  cy.intercept(
+    "GET",
+    "https://freesound.org/apiv2/search/text/?fields=id%2Cname%2Cpreviews%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings&page_size=10&query=hello",
```

Comment:
I needed to change this because Cypress was intercepting requests to `https://freesound.org/apiv2/search/text/?fields=id%2Cname%2Cpreviews%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings&page_size=10&page=2&query=hello`, which is the API request this application makes when moving onto page 2 of the search results for "hello", and it was showing the results for the first page of search results, instead, with the `search-results.json` fixture.